code/script.js

```diff
@@ -2,208 +2,236 @@
     eyes: 'blue',
     accessories: ['glasses', 'hat'],
-    other: ['smoker']
+    other: ['smoker'],
+    gender: ['male']
   },
   {
     name: 'Jeane',
     img: 'images/jeane.svg',
     hair: 'brown',
     eyes: 'green',
     accessories: ['glasses'],
-    other: []
+    other: [],
+    gender: ['female']
   },
   {
     name: 'Jed',
     img: 'images/jed.svg',
     hair: 'orange',
     eyes: 'green',
     accessories: ['glasses', 'hat'],
-    other: ['smoker']
+    other: ['smoker'],
+    gender: ['male']
   },
   {
     name: 'Jenni',
     img: 'images/jenni.svg',
     hair: 'white',
     eyes: 'hidden',
-    accessories: ['hat'],
-    other: []
+    accessories: ['hat', 'jewelry'],
+    other: [],
+    gender: ['female']
   },
   {
     name: 'Jeri',
     img: 'images/jeri.svg',
     hair: 'orange',
     eyes: 'green',
     accessories: ['glasses'],
-    other: []
+    other: [],
+    gender: ['female']
   },
   {
     name: 'Jerry',
     img: 'images/jerry.svg',
-    hair: 'hidden',
+    hair: 'hiddenhair',
     eyes: 'blue',
     accessories: ['hat'],
-    other: []
+    other: [],
+    gender: ['male']
   },
   {
     name: 'Jess',
     img: 'images/jess.svg',
     hair: 'black',
     eyes: 'blue',
     accessories: ['glasses'],
-    other: []
+    other: [],
+    gender: ['female']
   },
   {
     name: 'Jocelyn',
     img: 'images/jocelyn.svg',
     hair: 'black',
     eyes: 'brown',
-    accessories: ['glasses'],
-    other: []
+    accessories: ['glasses', 'jewelry'],
+    other: [],
+    gender: ['female']
   },
   {
     name: 'Jon',
     img: 'images/jon.svg',
     hair: 'brown',
     eyes: 'green',
     accessories: ['glasses'],
-    other: []
+    other: [],
+    gender: ['male']
   },
   {
     name: 'Jordan',
     img: 'images/jordan.svg',
     hair: 'yellow',
     eyes: 'hidden',
-    accessories: ['glasses', 'hat'],
-    other: []
+    accessories: ['sunglasses', 'hat'],
+    other: [],
+    gender: ['male']
   },
   {
     name: 'Josephine',
     img: 'images/josephine.svg',
     hair: 'grey',
     eyes: 'brown',
-    accessories: [],
-    other: []
+    accessories: ['jewelry'],
+    other: [],
+    gender: ['female']
   },
   {
     name: 'Josh',
     img: 'images/josh.svg',
     hair: 'yellow',
     eyes: 'green',
     accessories: [],
-    other: []
+    other: [],
+    gender: ['male']
   },
   {
     name: 'Jude',
     img: 'images/jude.svg',
     hair: 'black',
     eyes: 'green',
     accessories: [],
-    other: []
+    other: [],
+    gender: ['male']
   },
   {
     name: 'Julie',
     img: 'images/julie.svg',
     hair: 'black',
     eyes: 'brown',
     accessories: ['glasses', 'hat'],
-    other: []
+    other: [],
+    gender: ['female']
   },
 ]
 
 // Global variables
-let secret
-let currentQuestion
-let charactersInPlay
+let secret //will be the secret person object
+let currentQuestion //Will be the current question object
+let charactersInPlay //Will be an array of all people left in the game
```

Comment:
Its great that you have a lot of comments in your code! Makes it easy to understand!